src/main/scala/com/microsoft/hyperspace/index/IndexLogEntry.scala

```diff
@@ -608,7 +620,26 @@ case class IndexLogEntryTag[T](name: String)
 object IndexLogEntry {
   val VERSION: String = "0.1"
 
+  val HYPERSPACE_PROJECT_VERSION: String = "hyperspaceVersion"
+
   def schemaString(schema: StructType): String = schema.json
+
+  /*
+   * Default IndexLogEntry Properties
+   */
+  def apply(name: String,
+            derivedDataset: CoveringIndex,
+            content: Content,
+            source: Source
+           ): IndexLogEntry = {
```

Comment:
```suggestion
            source: Source): IndexLogEntry = {
```
Could you check scalafmt setting?